examples/fizzbuzz/Makefile

```diff
@@ -0,0 +1,24 @@
+TARGET := $(shell basename `pwd`).bin
+
+.PHONY: all clean setup
+
+BIN_DIR := bin
+CPP_SOURCE_DIR := cpp-src
+CPP_SOURCES := $(shell find $(CPP_SOURCE_DIR) -name "*.cpp")
+
+CC := clang++
+CCFLAGS := -std=c++20 -I$(CPP_SOURCES)
+
+MKDIR := @mkdir
+RM := @rm
+
+all: setup
+	g++ -I$(CPP_SOURCE_DIR) $(CPP_SOURCES) -o $(BIN_DIR)/$(TARGET)
```

Comment:
Oopsie! Fixed this (https://github.com/magnolia-lang/magnolia-lang/commit/c99ecea5836fb198464da99fba6a0d2735037378).